.github/workflows/a-team.yml

```diff
@@ -1,9 +1,9 @@
-name: ateam
+name: a-team
 
 on:
   pull_request:
     paths-ignore:
-      - '**.md'
+      - "**.md"
```

Comment:
@lioramilbaum  Not sure about the motive for this change, but at the official documentation it uses single quotes:
https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-ignoring-paths

Maybe using double quotes might create unexpected behaviors, like expanding the glob before to pass it to the `path-ignore`directive. I don't know. I'd rather leave it as single quotes, just in case.

Reply:
This change is done automatically by a vscode format plugin.  I will remove it.